packages/eslint-plugin-lit-a11y/lib/rules/accessible-emoji.js

```diff
@@ -49,8 +53,13 @@ const AccessibleEmojiRule = {
     }
 
     return {
+      ImportDeclaration(node) {
```

Comment:
can we implement this once and extend for rules, instead of copying?

Reply:
maybe on `context`? https://eslint.org/docs/developer-guide/working-with-rules#the-context-object

Reply:
We cant, because for each file we need to find the lit-html ImportDeclaration

Reply:
but surely we can just define this code elsewhere and import it

at the very least
```js
create(context) {
  return {
    ...checkLitHtmlImport,
   TaggedTemplateExpression(...)
  }
}
```

Reply:
I wanted to do something like that, but it gets messy because we need the `isLitHtml` variable from the scope of `create`